refinery/adapter/transformers.py

```diff
@@ -49,6 +53,9 @@ def build_classification_dataset(
     if os.path.exists(test_file_path):
         os.remove(test_file_path)
 
-    index = {"train": df_train[primary_keys], "test": df_test[primary_keys]}
+    index = {
+        "train": df_train[primary_keys].to_dict(orient="records"),
+        "test": df_test[primary_keys].to_dict(orient="records"),
```

Comment:
is this correct? the same value is assigned to train and test

Reply:
It is not the same :) It is `df_TRAIN` and `df_TEST`